packages/webstudio-sdk/src/css/categories.ts

```diff
@@ -0,0 +1,253 @@
+import type { StyleProperty } from "./types";
```

Comment:
It feels like this should be a separate package. @kof does this exist in designer somewhere?

Reply:
designer is using it from here, yeah, we will split a lot of things into separate packages